theseus/labs/lie/functional/so3_impl.py

```diff
@@ -542,21 +551,19 @@ def _forward_impl(cls, tangent_vector):
 
     @classmethod
     def setup_context(cls, ctx, inputs, outputs):
-        # inputs is (tangent_vector, )
-        ctx.save_for_backward(inputs[0])
+        pass
```

Comment:
This method is not abstract, so you can omit it from the class now (it's already defined in the base class).

Reply:
Removed.